### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by semihokur**

### CODE REVIEW

File: Documentation/Wiki/Release-Notes.md [hunk 1]
```diff
@@ -11,9 +11,26 @@ This page is a curated list of the release notes for releases after 0.20170619.4
...
+- Fix IPC pip ordering for BinaryGraph
+- CBdependencies feature an opt-out instead of opt-in
+- Improve time to first pip for distributed builds
+- Enforce weak fingerprint augmentation for MSBuild-scheduled pips
+- Use xattrs to mark shared opaque outputs on Mac
+
+# 0.1.0-20191003.4 (Release [42310](https://dev.azure.com/mseng/Domino/_releaseProgress?_a=release-pipeline-progress&releaseId=42310)). Released 10/09/2019
 - Removed old Microsoft.ContentStoreApp.exe from deployment (replaced with ContentStoreApp.exe)
-- ...
+- Proper SIGILL handling on macOS
```
Comment (almiliMSFT) [inline Documentation/Wiki/Release-Notes.md]: Sorry I forgot to do it.

Comment (semihokur) [reply Documentation/Wiki/Release-Notes.md]: it is not clear from our dri doc :) so, i updated the doc as well

Comment (almiliMSFT) [reply Documentation/Wiki/Release-Notes.md]: It was clear; I saw it the instructions, and I saw Oleksii do it the week before, so I was going to do it, but I just forgot.

**Pull request merged**